Trappist/src/Promact.Trappist.Web/wwwroot/app/questions/questions-dashboard/questions-dashboard.component.ts

```diff
@@ -50,11 +58,17 @@ export class QuestionsDashboardComponent implements OnInit {
         this.easy = 0;
         this.medium = 0;
         this.hard = 0;
-        this.loader = true;
+        this.isAllQuestionsHaveCome = false;
+        this.index = 0;
+        this.selectedDifficulty = DifficultyLevel['All'];
+        this.difficultyLevel = 'All';
+        this.categroyId = 0;
     }
 
     ngOnInit() {
-        this.getAllQuestions();
+        this.loader = true;
```

Comment:
no v cant put this in any method , it is for displaying the spinner when this component is called first not any method